database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -14,7 +14,6 @@ import cromwell.database.sql.tables.{
   MetadataEntry,
   WorkflowMetadataSummaryEntry
 }
```

Comment:
The changes to this file are a revert of https://github.com/broadinstitute/cromwell/pull/7225